src/components/SystemMessageNotification/SystemMessageNotification.module.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../theme";
+
+.warning{
+    border-radius: 0 !important;
+    background: #c60909;
+    text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
+    color: white;
+    padding-bottom: 1px;
+    margin-bottom: 0px;
+   /* position: absolute;
+    right: 0;
+    left:0;*/
+
+}
+
+.notice {
+    border-radius: 0 !important;
+    background: darkslategrey;
+    color: white;
+    padding-bottom: 2px;
+    margin-bottom: 0px;
+    /*position: absolute;
+    right: 0;
+    left:0;
+*/
+}
+
+.info {
+    border-radius: 0 !important;
+    background: #3871dc;
+    color: white;
+    padding-bottom: 2px;
+    margin-bottom: 0px;
+   /* position: absolute;
+    right: 0;
+    left:0;*/
+    }
```

Comment:
@AStiner  You are duplicating a lot of style between these message types. Create a `.alert` class to hold all the base styles and use the remaining types as only modifications (ex. background color).